cmd/broker/main.go

```diff
@@ -17,23 +18,24 @@ func main() {
 	logger := lager.NewLogger("blockhead-broker")
 	logger.RegisterSink(lager.NewWriterSink(os.Stdout, lager.DEBUG))
 
-	if len(os.Args) < 2 {
-		logger.Fatal("main", errors.New("config file is missing"))
+	if len(os.Args) < 3 {
+		logger.Fatal("main", errors.New("config file and/or service directory missing"))
 	}
 
 	configFilepath := os.Args[1]
-	cfg, err := config.NewConfig(configFilepath)
+	servicePath := os.Args[2]
+	state, err := config.NewState(configFilepath, servicePath, uuid.New)
```

Comment:
why cant we make `uuid.New` internal to State?

Reply:
The real reason was for test purposes because comparing maps is difficult if we don't have access to the uuid. I am working on some custom matchers that would get rid of this problem.